### ISSUE TITLE
Support for x86 android devices

### ISSUE THREAD

**Issue by meepingsnesroms** [openMSX/openMSX#1071]
On the website you only have an apk for android armv6.
armv7, armv8 and x86 are not supported.
Although armv7 and armv8 are not an issue because v7 just added more instructions and v8 has v7 emulation mode.

**Comment by matsp888** [comment]
I agree that it would be nice with a x86 build as well, since I have an ASUS MemoPad FHD10 which is Intel based.
